src/runtime/manifest-parser.peg

```diff
@@ -339,6 +339,16 @@ ParticleClaimStatement
   }
 
 ParticleClaimExpression
+  = first:ParticleClaim rest:(whiteSpace 'and' whiteSpace ParticleClaim)*
```

Comment:
There's an issue here, because ParticleClaimDerivesFrom also supports `and` expressions. I think it needs to be updated to only accept a single handle, like ParticleClaimIsTag does.

Reply:
Huh? Both rely on the handle being part of the ParticleClaimStatement, so both accept a single handle and multiple ParticleClaims. I.e. it already does what you want. Or am I missing something?